source/includes/debt-settlement.md.erb

```diff
@@ -587,31 +586,42 @@ Content-Type: application/json
-| investorRepayments                | array  | The repayments per investor.                                                   | required |
-| investorRepayments[].investmentId | string | The ID of the of the investment                                                | required |
-| investorRepayments[].type         | string | The type of the repayment Possible values are: <br>`CAPITAL`<br>`INTEREST`<br> | required |
-| investorRepayments[].amount       | ref    | The amount being repaid                                                        | required |
-| investorRepayments[].tax          | ref    | The amount of tax being withheld from this repayment                           | optional |
+| Name                              | Type   | Description                                                                                            | Required |
+| --------------------------------- | ------ | ------------------------------------------------------------------------------------------------------ | -------- |
+| reference                         | string | The payment reference used for the repayment of funds. This will be present on the wallet statement.   | required |
+| sourceWalletId                    | string | The wallet this distribution should be made from                                                       | required |
+| investorRepayments                | array  | The repayments per investor.                                                                           | required |
+| investorRepayments[].investmentId | string | The ID of the of the investment as specified in the original investment creation call                  | required |
+| investorRepayments[].type         | string | The type of the repayment Possible values are: <br>`CAPITAL`<br>`INTEREST`<br>                         | required |
+| investorRepayments[].amount       | ref    | The amount being repaid                                                                                | required |
+| investorRepayments[].tax          | ref    | The amount of tax being withheld from this repayment                                                   | optional |
 ### Response
 | Name                     | Type   | Description                                  |
 | ------------------------ | ------ | -------------------------------------------- |
 | repaymentBatchReference  | string | The unique ID for this repayment request     |
 
+### Possible HTTP Statuses 
+
+| HTTP Status Code | Description                                      | Body                                      | Content-Type     |                 
+|------------------|--------------------------------------------------|------------------------------------------ |------------------| 
+| 200 Ok           | Batch accepted to be instructed                  | See above                                 | application/json |                 
```

Comment:
Suggestions:

200 - Batch accepted to be instructed. Note that this does not indicate completion or success. Subsequent processing is asynchronous, so the caller should listen to the webhooks detailed above to determine progress.
50x - Unknown errors and timeouts. The caller should not assume that the request has failed. Listen for any subsequent webhooks indicating processing may have started, and ensure any automated retries use the `X-CLIENT-REQUEST-ID` header and repeat for a maximum of 24 hours to prevent double-processing